syntax = "proto3";

package helloMLBridge;

service helloMLBridgeService {
  rpc getTensor(ActionRequest) returns (TensorResponse) {}
  rpc getAdvice(TensorResponse) returns (ActionRequest) {}
}

message TensorResponse { repeated float tensor = 1; }

message ActionRequest { int32 action = 1; }
